streamr-docker-dev/bin.sh

```diff
@@ -22,7 +22,7 @@ HELP=0
 
 SERVICE_COMBINATIONS=(
 ""
-"cassandra kafka init_keyspace mysql redis smtp zookeeper nginx broker data-api"
+"cassandra kafka init_keyspace mysql redis smtp zookeeper nginx broker"
```

Comment:
Why this change? In the docs we recommend to use this option (option `1`) when developing `engine-and-editor`, and `data-api` is needed for that

Reply:
Are you talking about backend or frontend development?

Reply:
Backend

Reply:
Hmm, I think that it occurs mostly in frontend development, true

Reply:
After consulting with Eric it can affect the backend development.